test/ElvClient.test.js

```diff
@@ -284,7 +284,8 @@ describe("Test ElvClient", () => {
   });
 
   describe("Content Libraries", () => {
-    test("Set Tenant ID For User", async () => {
+    test.skip("Set Tenant ID For User", async () => {
+      // TODO require tenantContractId set for tenant group ID
```

Comment:
@elv-kevin trying to deploy `baseTenantSpace` contract, how do I get the bytecode? From `src/contracts/v3/baseTenantSpace.js` only able to get the abi. Not able to find any example code. 

In elv-live-js, we have contracts abi and bin stored as separate file.

Reply:
client doesn't have bytecode because it's not used. If you need it for tests, you can compile it elsewhere and add it to the test directory for import

Reply:
thanks will do.